wordpress-mobile/WordPress-iOS
# [Site creation] Style site information

Implements #10481 

Styling of the Site Info screen.

A few screenshots: 

iPhone SE:
![simulator screen shot - iphone se - 2018-11-26 at 14 43 32](https://user-images.githubusercontent.com/2722505/48997468-074c7700-f18b-11e8-9d9b-63545d164d90.png)

iPhone X:
![simulator screen shot - iphone x - 2018-11-26 at 14 45 31](https://user-images.githubusercontent.com/2722505/48997463-03b8f000-f18b-11e8-8115-b20c5fceb41c.png)
![simulator screen shot - iphone x - 2018-11-26 at 14 45 36](https://user-images.githubusercontent.com/2722505/48997510-2ea34400-f18b-11e8-95c2-eaea1bdeb067.png)
![simulator screen shot - iphone x - 2018-11-26 at 14 45 55](https://user-images.githubusercontent.com/2722505/48997525-3d89f680-f18b-11e8-91e7-34dcce47760a.png)

iPad Pro 11" (2018)
![simulator screen shot - ipad pro 11-inch - 2018-11-26 at 14 47 18](https://user-images.githubusercontent.com/2722505/48997548-4ed30300-f18b-11e8-8554-5c1825a18f3e.png)
![simulator screen shot - ipad pro 11-inch - 2018-11-26 at 14 47 22](https://user-images.githubusercontent.com/2722505/48997564-57c3d480-f18b-11e8-856f-a30bc138fd98.png)

To test:
- Check out the branch
- Navigate to My Sites > + > Enhanced site creation
- Navigate to the Site Info Screen
- The label and background color of the Skip button should change when entering text in either textfield

@SylvesterWilmott could you please take a look as well, as this needs design review?
